src/block/block_write.c

```diff
@@ -181,7 +181,9 @@ __wt_block_write_size(WT_SESSION_IMPL *session, WT_BLOCK *block, size_t *sizep)
      * interested in debugging corner cases anyway.
      */
     *sizep = (size_t)WT_ALIGN(*sizep + WT_BLOCK_HEADER_BYTE_SIZE, block->allocsize);
-    return (*sizep > UINT32_MAX - 1024 ? EINVAL : 0);
+    if (*sizep > UINT32_MAX - 1024)
+        WT_RET_MSG(session, EINVAL, "requested block size too large");
```

Comment:
Grammatical nit: "block size too large" --> "block size is too large"

Reply:
Fixed in fb83093f8bb8912a438f7fcc4a987b70034a86b5